src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -62,9 +63,9 @@ static void Main(string[] args)
             TestRunSystem = ActorSystem.Create("TestRunnerLogging");
             SinkCoordinator = TestRunSystem.ActorOf(Props.Create<SinkCoordinator>(), "sinkCoordinator");
 
-            var assemblyName = args[0];
-
+            var assemblyName = Path.GetFullPath(args[0]);
```

Comment:
We now capture the full path of the assembly we're testing and pass that to the individual NodeTestRunner processes